Bigtable/tests/System/BigtableTestCase.php

```diff
@@ -25,6 +25,7 @@
 use Google\Cloud\Bigtable\Admin\V2\Table;
 use Google\Cloud\Bigtable\BigtableClient;
 use Google\Cloud\Core\Testing\System\SystemTestCase;
+use Google\Cloud\Bigtable\Admin\V2\StorageType;
```

Comment:
nit: improperly sorted imports.

Reply:
Oh, so we sort it according to last name of the import. I thought it was sorting by comparing the whole import string , my bad : (

Thanks for the suggestion.